modern/org/commcare/modern/session/SessionWrapper.java

```diff
@@ -59,4 +63,10 @@ public void clearVolitiles() {
     public void setComputedDatum() {
         setComputedDatum(getEvaluationContext());
     }
+
+    public FormDef loadFormByXmlns(String xmlns) {
```

Comment:
can you add a `@SuppressWarning("unused")` annotation so that we know this is used somewhere.
Also, this exact method is in `CommCareConfigEngine` and can be made static, so you should consider just calling that method or moving it here.